Tools/Scripts/prepare-ChangeLog

```diff
@@ -2302,21 +2133,7 @@ sub generateFileList(\%$$$\%)
         my $original;
         my $file;
 
-        if (isSVN()) {
-            my $matches;
-            $matches = /^([ ACDMR])([ CM]).{5} (.+?)[\r\n]*$/;
-            $status = $1;
-            $propertyStatus = $2;
-            $file = $3;
-            if ($matches) {
-                $file = normalizePath($file);
-                $original = findOriginalFileFromSvn($file) if substr($_, 3, 1) eq "+";
-                my $isAdd = isAddedStatus($status);
-                $propertyChanges = determinePropertyChanges($file, $isAdd, $original) if isModifiedStatus($propertyStatus) || $isAdd;
-            } else {
-                print;  # error output from svn stat
-            }
-        } elsif (isGit()) {
+        if (isGit()) {
```

Comment:
Most `isGit()` calls should probably be removed, but there may be one or two that sanity check the directory.